docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -627,9 +596,78 @@ simplifying assumptions that we have included:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+New simulants are added by migration into the US from other countries.
+We simulate two kinds of immigration: household moves and non-reference-person moves.
+
+#. A **household move** is when an entire household (which may be a single person) enters from outside the country as a unit,
+   preserving relationships within the unit.
+#. A **non-reference-person move** is when a person enters from outside the country and joins an existing household.
+   Non-reference-person moves are independent, single-person events that do not preserve relationship structure.
+
+For the purposes of immigration, a group quarters person should be treated identically to a one-person household,
+with the group quarters person as the reference person.
+This means that group quarters people should only enter the US in a household move.
+
+The number of simulants who move to the US each year is informed by the ACS' "residence one year ago" question.
+Specifically, a value of 2 for variable :code:`MIG` indicates that a respondent lived outside the US one year ago,
+while any other value indicates that they lived within the US.
+We refer to respondents who were not living in the United States one year ago as "recent immigrants."
+Our assumption is that the number (and characteristics) of recent immigrants
+in the 2019 ACS will be replicated in each future year.
+
+We also assume that the proportions of recent immigrants by move type (household or non-reference-person) will remain constant.
+Though in reality not all moves into the US follow one of these two patterns, we assume that any new immigrant in a household
+where the reference person is also a new immigrant was part of a household move, while any new immigrant in a household where the
+reference person is not a new immigrant moved to the US in a non-reference-person move.
+
+Specifically, the yearly rate at which simulants are added to the population by each move type is given by
+the (weighted) proportion of ACS PUMS persons that are recent immigrants consistent with that move type.
+At each time step:
+
+#. PUMS households with reference people who are recent immigrants, after removing household members who are not recent immigrants,
+   are sampled with replacement until the desired number of simulants added in household moves is reached.
+#. PUMS recent immigrants not living in such households are sampled with replacement until the desired
+   number of simulants added in non-reference-person moves is reached.
+
+Households added in a household move **that are not GQ people** are assigned a new address and household ID so as not
+to conflict with the existing household that was sampled to create them.
+GQ people added as a single-person household should retain the GQ type from sampling.
+
+Simulants added by a non-reference-person move join a randomly-selected existing household matching their PUMA.
```

Comment:
currently the PUMA never changes after initialization. are we concerned about the likelihood that this causes us to add PERSON A in the data to the household they are already in? (but i guess... this would be a younger version of themselves, because the PERSON A from initialization would have aged a bit?)

i think this is probably quite unlikely, but am not sure how large PUMAs tend to be

Reply:
PUMAs are 100,000+ population, which means ~5,000+ ACS respondents in a 5-year file. That would make this pretty unlikely.

That said, the fact that PUMAs never change after initialization (i.e. at the time of domestic migration) is a problem. It is not correct to be doing this matching on PUMA when it may no longer represent where a household "should" be living. We will need to update the domestic migration component to correct this, and are working out the best way to do so.

Reply:
I did not note this in the documentation (I will add it), but of course, simulants added via migration should get a new unique simulant ID for PRL tracking purposes (they wouldn't literally be the same people, even if added to the same household).